prow/git/git.go

```diff
@@ -50,7 +49,7 @@ type Client struct {
 	dir string
 	// git is the path to the git binary.
 	git string
-	// base is the base path for git clone calls. For users it will be set to
+	// bases are the base paths for git clone calls. For users it will be set to
 	// GitHub, but for tests set it to a directory with git repos.
 	base string
```

Comment:
did you mean to make this a slice?

Reply:
Yep, on another branch I have been testing multiple gitEndpoints. Nothing promising yet though.

Reply:
I'll revert this comment for now.